packages/genesys-spark-components/src/components/stable/gux-form-field/gux-form-field.types.ts

```diff
@@ -1,3 +1,3 @@
 export type GuxFormFieldLabelPosition = 'above' | 'beside' | 'screenreader';
 
-export type GuxFormFieldIndicatorMark = 'required' | 'optional';
+export type GuxFormFieldIndicatorMark = 'required' | 'optional' | 'none';
```

Comment:
I added this new variant to make is easier to follow the Spark guidelines

https://spark.genesys.com/latest/components/forms-and-inputs/form-control-primitives/form-field-label-v-l98Yn3K5#search-678721a0-b3f6-11ee-b9e6-615a7d5c6ec1